easybuild/easyconfigs/p/Paraview/Paraview-4.3.1-intel-2015a-Python-2.7.10-mpi.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'CMakeMake'
+
+name = 'Paraview'
+version = '4.3.1'
+
+homepage = "http://www.paraview.org"
+description = "Paraview is a scientific parallel visualizer."
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % download_suffix]
+sources = ["ParaView-v%(version)s-source.tar.gz"]
+
+patches = ['missingheader-%(version)s.patch']
+
+python = 'Python'
+pyver = '2.7.10'
+versionsuffix = '-%s-%s-mpi' % (python, pyver)
```

Comment:
because Python is a very common dep, it helps users identifying which modules are compatible with each other

the others are far less common, so listing everything in the versionsuffix yields no added benefit, it just makes things uglier

it's a pragmatic choice